pkg/bootstrap/types.go

```diff
@@ -12,6 +12,10 @@ type KuryrBootstrapResult struct {
 	PodSecurityGroups []string
 	ClusterID         string
 	OpenStackCloud    clientconfig.Cloud
+	CA                string
```

Comment:
How about renaming it to WebhookCA and WebhookCAKey to make sure it's not confused with CA Kuryr uses to connect to OpenStack API?